File: js/phenolist2tsv.js
const fs = require('fs')

const phenolist = JSON.parse(fs.readFileSync(process.argv[2], 'utf8'))

console.log("phenocode\tname\tcategory\tnum_cases\tnum_controls\tgc_lambda")
phenolist.forEach(pheno => {
    console.log(pheno.phenocode + '\t' + pheno.phenostring + '\t' + pheno.category + '\t' + pheno.num_cases + '\t' + pheno.num_controls + '\t' + pheno.gc_lambda['0.5'])
})

File: scripts/phenolist2tsv.py
import json
import sys

phenos = json.load(open(sys.argv[1], 'rt'))
print('\t'.join(['phenocode','name','category','num_cases','num_controls','gc_lambda','num_gw_significant']))
for pheno in phenos:
    print('\t'.join([pheno['phenocode'], pheno['phenostring'], pheno['category'], str(pheno['num_cases']), str(pheno['num_controls']), str(pheno['gc_lambda']['0.5']), str(pheno['num_gw_significant'])]))
